cypress/integration/dashboard.spec.js

```diff
@@ -1,6 +1,6 @@
 describe('When a user first accesses the Dashboard', () => {
   beforeEach('setup stubs and visit Dashboard', () => {
-    cy.setDataStubs()
+    cy.setReadStubs()
```

Comment:
I like how you made this into a function call with the stubs, helps with the readability!